### PR TITLE
Fix ACS swagger spec error in array definition

### PR DESCRIPTION
**Pull request by lmazuel**
Py release 3.0.1

### CODE REVIEW

File: azure-mgmt-containerservice/azure/mgmt/containerservice/models/orchestrator_version_profile_list_result.py [hunk 1]
```diff
@@ -40,10 +40,11 @@ class OrchestratorVersionProfileListResult(Model):
         'id': {'key': 'id', 'type': 'str'},
         'name': {'key': 'name', 'type': 'str'},
         'type': {'key': 'type', 'type': 'str'},
-        'orchestrators': {'key': 'properties.orchestrators', 'type': 'OrchestratorVersionProfile'},
+        'orchestrators': {'key': 'properties.orchestrators', 'type': '[OrchestratorVersionProfile]'},
```
Comment (lmazuel) [inline azure-mgmt-containerservice/azure/mgmt/containerservice/models/orchestrator_version_profile_list_result.py]: @mboersma I guess it's what you want?

**Comment by lmazuel** [comment]
@mboersma LGTM. Most of the update is Autorest.Python 2.1.34, and I see your array fix. Once CI for Python is green, I can merge and release both PR.

File: azure-mgmt-containerservice/azure/mgmt/containerservice/models/orchestrator_version_profile_list_result.py (hunk 1 above)
Comment (mboersma) [reply azure-mgmt-containerservice/azure/mgmt/containerservice/models/orchestrator_version_profile_list_result.py]: @lmazuel yes, that looks like the important change that I've been testing with locally. 👍

**Comment by lmazuel** [comment]
@mboersma The release CI job is started, should be in PyPI in 10 minutes

**Comment by mboersma** [comment]
Excellent, thanks so much for your help!

**Pull request merged**